### PR TITLE
Added support for doctrine migrations 3

### PR DESCRIPTION
**Pull request by edefimov**
This PR adds support for doctrine migrations v3 and keeping compatibility with v2.

### CODE REVIEW

**Comment by cgrabenstein** [comment]
@edefimov the linked PR was just merged.

**Comment by edefimov** [comment]
@kbond, PR is ready

File: composer.json [hunk 1]
```diff
@@ -23,18 +23,18 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
```
Comment (kbond) [inline composer.json:26]: Can you leave as is for now. We'll do this is #251

File: composer.json [hunk 2]
```diff
@@ -23,18 +23,18 @@
...
         "symfony/framework-bundle": "^3.4 || ^4.0 || ^5.0",
-        "laminas/laminas-diagnostics": "^1.5"
+        "laminas/laminas-diagnostics": "^1.8"
```
Comment (kbond) [inline composer.json:28]: Can you leave as is. We need to support PHP 7.1 for now... This will fix the test failure.

**Review by kbond — CHANGES_REQUESTED** [review]
Looks good, just a few dependency changes.

**Review by kbond — APPROVED** [review]

**Comment by kbond** [comment]
Thanks @edefimov!

**Pull request merged**